### PR TITLE
New translations for the UI

### PR DESCRIPTION
**Pull request by kelson42**
Should offer a few new strings translations

### CODE REVIEW

**Comment by soloturn** [comment]
@kelson42 would be great if you could put the texts of ticket, pull request into the commit comment. currently it is necessary to open all three to understand why you make the commit, what you reach, and what your next step is. while the info is there, it takes more time to find it. also it is missing how you did it so one reading the commit can do it by her own next time.

to give an example:
resync translations to get newest strings, from ??

in #1263 a user has the phone set to default language french, set kiwiki to arabic, which then resulted in a mixed display: english in menu and description, french in description, arabic in filenames and sizes. so resync via xxx from whatebber.

**Comment by kelson42** [comment]
@soloturn This is a routine/maintenance operation, the information are in the development documentation on the wiki. If I put #1263 in the description it will close it and I don't want to close it as I believe this is not the only problem.

**Comment by soloturn** [comment]
putting the ticket number into the commit message does nothing, adding a keyword like "fixes" will do this:
https://help.github.com/en/articles/closing-issues-using-keywords

File: app/src/main/res/values-af/strings.xml [hunk 1]
```diff
@@ -31,4 +31,26 @@
   <string name="pref_language_chooser">Kies \'n taal</string>
   <string name="menu_bookmarks_list">Boekmerke</string>
   <string name="zim_manager">Biblioteek</string>
-</resources>
+  <plurals name="books_count">
```
Comment (macgills) [inline app/src/main/res/values-af/strings.xml]: This needs to be removed in every strings.xml, the project cannot compile

**Comment by kelson42** [comment]
@macgills Can we merge?

**Pull request merged**